libs/chlorophyll/scss/components/chips/_mixins.scss

```diff
@@ -0,0 +1,198 @@
+@mixin global {
```

Comment:
I like this pattern of grouping variables in a mixin! Maybe we should have a naming convention for variable mixins? Like `global-vars`, `prop-vars`, `icon-vars`, etc, in this example. That way, when you read the code which calls the mixin, you can easily tell that the `@include` statement is only outputting variables.

In a similar fashion, the style mixins in this example could be named `state-styles`, `variant-styles`, etc.

What do you think?